Added DelaunaySparse to package list

thchang opened the pull request:
DelaunaySparse is an open source Fortran 2003 package, recently published in ACM TOMS (Algorithm 1012) for performing piecewise linear interpolation via a sparse subset of the Delaunay mesh in high-dimensional spaces.  The publication process was subject to peer-review, not only of the paper, but also the source code, which is also published in the collected algorithms of ACM TOMS.  The code itself is open source (MIT license).  This code has been in development for almost 4 years and tested on thousands of real-world and synthetic data sets.  It is in a stable condition and in usage at several national labs and universities.  Features robust error-handling and C + Python interfaces.  Unfortunately, we do not have "5 stars" yet on Github.  Please consider including anyway.

certik commented:
I just gave you a star. @milancurcic, @everythingfunctional, @LKedward can one of you star it also? ;)

@thchang thanks for the submission! We would like to see more codes like these, peer reviewed with a publication. Fortran is a great language precisely for applications like this one.

thchang commented:
Thanks all for the stars --- happy to see you guys addressing some of the issues in Fortran (lack of package manager, std libs, and performance portability), which will help keep old Fortran alive and relevant!  Also congrats on approval for GSoC!  I have other Fortran software under review for publication, so you'll probably hear from me again in the future -- will also recommend this to all other Fortran programmers I know

thchang commented:
sorry for close and reopen, I'm still learning how to github web interface...

certik commented:
@thchang thank you for submitting and for recommending us to your colleagues. Your code is now available at:

https://fortran-lang.org/packages/numerical

If there is any issue that you do not see addressed by some effort at Fortran-lang, please let us know! Looking forward to seeing you around.
